sklearn/pipeline.py

```diff
@@ -218,6 +223,12 @@ def _final_estimator(self):
 
     # Estimator interface
 
+    def _fit_single_transform(self, transformer, name, idx_transform, X, y,
+                              **fit_params_trans):
```

Comment:
I would write here (in a comment or in a docstring) that this method is really useful for subclassing. Elsewhere someone like me wonders why it's there.